charts/testkube-operator/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: "kubeshop/testkube-operator"
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
is this consistent with our pull policy for other images?

Reply:
IMHO we use versioned releases, so I don't see a good reason everytime to download the Docker image from Docker repository instead of using the cached version.

I can revert if our internal convention is for pullPolicy to be Always

Reply:
I think we should define it first ;) 

Just IMO default pull policy is good enough for our use cases so we could totally omit this field

FYI https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting

Reply:
@exu 
Should we keep it in the Helm values but with a blank value? 
`pullPolicy: ""`